x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_instances.tsx

```diff
@@ -40,13 +40,21 @@ export const alertInstancesTableColumns = (
     sortable: false,
     truncateText: true,
     'data-test-subj': 'alertInstancesTableCell-instance',
+    render: (value: string) => {
+      return (
+        <EuiToolTip anchorClassName={'eui-textTruncate'} content={value}>
+          <span>{value}</span>
+        </EuiToolTip>
+      );
+    },
   },
   {
     field: 'status',
     name: i18n.translate(
       'xpack.triggersActionsUI.sections.alertDetails.alertInstancesList.columns.status',
       { defaultMessage: 'Status' }
     ),
+    width: '100px',
```

Comment:
Are we supposed to be using hard coded pixel dimensions? 🤔 
What are our guidelines on responsive design?
Any thoughts from @elastic/kibana-design on this?

Reply:
Table columns are an exception to that otherwise hard rule. This is fine, thanks for checking!